### PR TITLE
Summary of "Getting All Assignments"

### PR DESCRIPTION
**Pull request by cpreid**
Summary of "Getting All Assignments"

### CODE REVIEW

**Comment by jasonwzm** [comment]
🚀 Deployed on https://66bf9ce2f1bcf788f30d5f29--cozy-fox-0defec.netlify.app

**Review by tore-statsig — APPROVED** [review]
good start - should probably have a note that it will hash by default, but refer to language docs on how to disable hashing for that method

still need the full docs in each language for how to disable hashing (my team can handle that)

File: docs/server/concepts/all_assignments.mdx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+:::info
+Our SDKs are designed to be invoked in your code at the point where you're serving the experiment treatment or feature. 
+For this reason, we recommend leveraging the standard approach of calling `getExperiment`, `checkGate`, or `getLayer` supplying the user object and the key as needed to determine what to serve. 
+:::
+
+Often times, customers have the desire to capture all assignments (experiment, gates) in a single call on the server for a each user at the start of their session.
+This practice is commonly done as a means to pass assignment information to the client-side a single time, in order to inform the client app of the test groups and features it should apply.
+
+For this use-case, we recommend using the [Client Bootstrapping](/client/concepts/bootstrapping) approach via the `getClientInitializeResponse` server sdk call to generate the full assignments payload (note that the experiment and feature keys within are hashed for privacy) that gets passed down to the client SDK.
```
Comment (tore-statsig) [inline docs/server/concepts/all_assignments.mdx:15]: I dont think id call it the "client bootstrapping approach.

But should add a note on how this method is intented to bootstrap our client sdks (so thats what dictates the format), and therefore it also hashes names by default

Comment (cpreid) [reply docs/server/concepts/all_assignments.mdx]: updated it using this language

**Pull request merged**